src/sky.cpp

```diff
@@ -537,6 +581,44 @@ void Sky::render()
 			}
 			driver->drawIndexedTriangleFan(&vertices[0], 4, indices, 2);
 		}
+
+		// Draw upper blend fog band
+		for (u32 j = 0; j < 2; j++) {
+			video::SColor c = m_bgcolor_alpha;
+			video::SColor c_blend = m_skycolor_alpha; //
+				vertices[0] = video::S3DVertex(-1, -0.1, -1, 0, 0, 1, c, t, t);
+				vertices[1] = video::S3DVertex( 1, -0.1, -1, 0, 0, 1, c, o, t);
+				vertices[2] = video::S3DVertex( 1, 0.3, -1, 0, 0, 1, c_blend, o, o);
+				vertices[3] = video::S3DVertex(-1, 0.3, -1, 0, 0, 1, c_blend, t, o);
```

Comment:
Why does this rise to 0.3? Now the sun and moon will start to disappear while still high in the sky and nowhere near the horizon.

Reply:
It doesn't affect the sun and moon. It's entire premise is to blend the alpha where the sun dips below a false horizon.

Reply:
Sorry my mistake, both colours have alpha 0.